src/solution/my_edge_app/docker-compose.yml

```diff
@@ -33,27 +33,33 @@ services:
       - grafana                         # Wait for start of container 'grafana'
 
 ##### INFLUXDB ######
+  init-db:
+    image: init-db:v0.0.1               # Docker Init Container to Setup the Main InfluxDB v2.2 Container
```

Comment:
Please add build statement with a path https://docs.docker.com/compose/compose-file/compose-file-v2/#build 
the goal is that user of the guide can just directly run docker-compose up -d commad after runnign the  [mosquite docker copose file](src/solution/mqtt_broker_mosquitto/docker-compose.yml)